ansible/roles/apache/templates/etc/apache2/conf-available/acme-challenge.conf.j2

```diff
@@ -0,0 +1,19 @@
+{# Copyright (C) 2014-2017 Maciej Delmanowski <drybjed@drybjed.net>
+ # Copyright (C) 2015-2017 Robin Schneider <ypid@riseup.net>
+ # Copyright (C) 2014-2017 DebOps <https://debops.org/>
+ # Copyright (C) 2020-2020 Sebastian Vaisov <sebastian@vaisov.lt>
+ # SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Please remove myself from it. I would also propose to remove Maciej. You are actually the only author of that file :)

Reply:
Fixed